.circleci/config.yml

```diff
@@ -1598,6 +1747,7 @@ jobs:
             cd /swirlds-platform;
             sed -i -e 's/github.com/services-jrs-regression/g' .gitmodules;
             git submodule update --init --recursive --checkout;
+            cd regression/; git checkout -b 02051-split-ci-jobs origin/02051-split-ci-jobs; git pull; cd -          
```

Comment:
This command seems complicated and hard to understand. It should be as simple as
`cd regression; git checkout 02051-split-ci-jobs; cd -;`

Most importantly, I think you missed the `;` at the end.

Reply:
I checked the [compiling log](https://app.circleci.com/pipelines/github/hashgraph/hedera-services/32601/workflows/1ccf5324-0b1d-4372-b274-639b953e14cd/jobs/108177/parallel-runs/0/steps/0-105)

It seems the last semicolon, is optional , it just like shell command , need it between command, but
the last one don't have to need it.

Reply:
hmm, interesting, I thought only the last command is allowed that, now it seems that the last sub-command of each command is also allowed.